### ISSUE TITLE
TOF plexiglass insert compatibility with DIRC?

### ISSUE THREAD

**Issue by jrstevenjlab** [JeffersonLab/hdds#49]
In the commit below the plexiglass shielding on the TOF beam hole was inserted and the DIRC commented out  (BTW, this change doesn't appear to be associated with a pull request, but was pushed directly to the master... so no one had a chance to review it).

https://github.com/JeffersonLab/hdds/commit/56d899925b93d27397a1b2ae37afd18a635b69ff

Why are these two elements of the geometry incompatible?  I went down to the hall this morning and don't see any conflict between the plexiglass shielding and the DIRC when it's installed.  I'm not aware of a plan to remove the plexiglass shielding on the TOF before we start the DIRC commissioning this Spring.

Richard, if there is an overlap in the geometry, please let us know where and if needed we can adjust the DIRC to avoid that.  We'll have a final survey at the end of this week to get a final as-built location for the DIRC which we'll then use to update the XML before the run.

**Comment by rjones30** [comment]
Hello Justin,

That was what I heard from Sean / Ashley, that the DIRC would play the role
of shielding the TOF and make this shielding block no longer needed. Is
this incorrect?

It was a mistake that I pushed directly to the master branch. I caught my
error after I had pressed Enter.

-Richard Jones

**Comment by jrstevenjlab** [comment]
Hi Richard,

Well, the DIRC is certainly some material upstream of the TOF which may "shield" it, but this is the first I've heard of plans to remove it before the run starts.  Anyway, that's a topic for a different forum.

For HDDS, can we confirm that there is no overlap between the plexiglass shielding and the DIRC.  This way I can add the DIRC back to the master for commissioning?

-Justin

**Comment by rjones30** [comment]
Justin,

As far as I know, yes. The only word I had that they were incompatible was
by email (don't remember who). If you can verify visually that they do not
interfere then go ahead and remove the comment about them being
incompatible and submit the new pull request. I will execute it.

-Richard J.

**Comment by jrstevenjlab** [comment]
From visual inspection in the hall the plexiglass doesn't stick out further than the maroon frame holding the scintillator bars and PMTs, so I don't see how it can interfere with the DIRC in reality.  If you or Maria could check for an overlap in the geometry that would be helpful to verify we don't have something mis-aligned in the simulation.

I'll submit a pull request to put the DIRC back into the geometry by default.

-Justin

![img_20190121_075546748](https://user-images.githubusercontent.com/13242886/51498701-904feb00-1d95-11e9-81b3-57414cc4c66a.jpg)

**Comment by pomm** [comment]
Hi,

The volume of the TOF, which I guess was the topic of discussion is called SHLH in the TOF geometry.
I’ve checked the master branch of hdds with master branches of hdgeant4, halld_sim, and halld_recon and DIRC on, and I do not see any overlaps for the whole GlueX detector!
That’s impressing. Thanks to Richard and all who corrected the overlaps!

Best regards,
Maria


On Jan 21, 2019, at 4:07 PM, Justin Stevens <notifications@github.com<mailto:notifications@github.com>> wrote:


From visual inspection in the hall the plexiglass doesn't stick out further than the maroon frame holding the scintillator bars and PMTs, so I don't see how it can interfere with the DIRC in reality. If you or Maria could check for an overlap in the geometry that would be helpful to verify we don't have something mis-aligned in the simulation.

I'll submit a pull request to put the DIRC back into the geometry by default.

-Justin

[img_20190121_075546748]<https://user-images.githubusercontent.com/13242886/51498701-904feb00-1d95-11e9-81b3-57414cc4c66a.jpg>

**Comment by zihlmann** [comment]
Hi All,

there is no plan to remove the plexiglas shielding in front of the TOF. 
However, if the special shielding test planed by
the CPP experiment will happen then the plexiglass shielding needs to be 
removed during that test.
That test wants to put a lead shielding in front of the TOF starting 
with a 1inch thickness. This will require the plexiglass
to be removed first (for simplicity of the mechanics).

cheers,
Beni

**Comment by jrstevenjlab** [comment]
Ok, then we will leave it in when the DIRC is attached to the downstream platform and it can be removed during the change to CPP.
-Justin

**Comment by faustus123** [comment]
Hi All,

  I don’t think we will actually remove it for CPP. We will install at most 2 r.l. of lead shielding which is
only ~1.2cm. The platform will be pulled back some and the there will be space to hang the additional
shielding upstream of the plexiglass. The CPP test is motivated by high TOF rates with the plexiglass
in so we want to supplement it with the lead.


Regards,
-David

-------------------------------------------------------------
David Lawrence Ph.D.
Staff Scientist, Thomas Jefferson National Accelerator Facility
Newport News, VA
davidl@jlab.org<mailto:davidl@jlab.org>
(757) 269-5567 W
(757) 746-6697 C


On Jan 22, 2019, at 8:14 AM, Justin Stevens <notifications@github.com<mailto:notifications@github.com>> wrote:

Ok, then we will leave it in when the DIRC is attached to the downstream platform and it can be removed during the change to CPP.
-Justin

**Comment by sdobbs** [comment]
OK, can we have two tagged versions of the geometry with the plexiglass shield, one with the DIRC and one without?

**Comment by rjones30** [comment]
Sean,

Yes, certainly. Can you recommend names for the two tags. I suggest that
one of them should be "default".

-Richard
